<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studata_display</class>
 <widget class="QWidget" name="studata_display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>461</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>302</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>请选择排序方式：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="fieldComboBox">
      <item>
       <property name="text">
        <string>学号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>姓名</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>分数</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="orderComboBox">
      <item>
       <property name="text">
        <string>升序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>降序</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存为文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
